.vscode/settings.json

```diff
@@ -31,5 +31,8 @@
     "Validate Solidity Definition": ["crates/solidity/inputs/language/definition/**/*.yml"]
   },
   "typescript.tsdk": "node_modules/typescript/lib",
-  "yaml.validate": false // Disable LSP validation for YAML files, as it is handled by our own cargo tasks
+  "yaml.validate": false,
+  "files.readonlyInclude": {
```

Comment:
nit: I suggest keeping this file alphabetically sorted.
This also breaks the comment before it.

Reply:
Did this through vscode settings. Changed

Reply:
I think it is still not sorted :( 
Not a blocker in any way, but it prevents accidentally adding these things twice, causing a conflict (#566 for example).